res/css/views/dialogs/_LocationViewDialog.scss

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_LocationViewDialog_wrapper .mx_Dialog {
+    padding: 0px;
+
+    /* Unset contain and position to allow the close button
+    to appear outside the dialog */
+    contain: unset;
+    position: unset;
+}
+
+.mx_LocationViewDialog {
+    /* subtract 0.5px to prevent single-pixel margin due to rounding */
+    width: calc(79vw - 0.5px);
+    height: calc(79vh - 0.5px);
```

Comment:
why 79% of the viewport? seems very arbitrary

Reply:
mx_Dialog has `max-height: 80%` so I went for 1 less to prevent overflow.

Reply:
why would it overflow if there's no paddings/margins?

Reply:
Thank you - whatever problem I had is not happening any more, so updated to 80 in #20417